tests/ttnn/unit_tests/operations/test_all_gather_matmul.py

```diff
@@ -107,10 +110,10 @@ def run_all_gather_matmul_on_t3000_impl(
     )
 
     ##### Perform the torch ops #####
-    matmul_output = torch.chunk(torch.matmul(input_tensor, weights_tensor), num_devices, dim)
+    matmul_output = torch.matmul(input_tensor, weights_tensor)
 
     ##### Perform the TT ops #####
-    for i in range(num_iters):
+    def run_op():
         # # all_gather
```

Comment:
Can delete some of these stale code snippets that are commented out.

Reply:
The commented out all gather/matmul are used for baseline perf measurements. ig I could put it under a global toggle variable to clean it up